<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaletteShowDialog</class>
 <widget class="QDialog">
  <property name="windowTitle">
   <string>Patch Dungeon</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout" name="imageHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pathComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="palGraphicsView">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>360</height>
      </size>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel">
       <property name="minimumWidth">
        <number>48</number>
       </property>
       <property name="text">
        <string>Zoom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOutButton">
       <property name="minimumWidth">
        <number>24</number>
       </property>
       <property name="maximumWidth">
        <number>24</number>
       </property>
       <property name="text">
        <string notr="true">-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEditWidget" name="zoomEdit">
       <property name="charWidth">
        <number>4</number>
       </property>
       <property name="alignment">
        <enum>Qt::AlignCenter</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomInButton">
       <property name="minimumWidth">
        <number>24</number>
       </property>
       <property name="maximumWidth">
        <number>24</number>
       </property>
       <property name="text">
        <string notr="true">+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
